tutordiscovery/templates/discovery/apps/settings/partials/common.py

```diff
@@ -21,7 +21,8 @@
 }
 
 ELASTICSEARCH_DSL['default'].update({
-    'hosts': "http://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}/"
+    {% if ELASTICSEARCH_SCHEME == "https" %}"use_ssl": True,{% endif %}
+    'hosts': {% if ELASTICSEARCH_SCHEME == "https" %}"https","http"{% endif %}://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}/"
```

Comment:
This line is going to be rendered as:

```
'hosts': "https","http"://elasticsearchhost:9200/"
```

which is not going to work.

I recommend instead:

```
'hosts': "{{ ELASTICSEARCH_SCHEME }}://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}/"
```

Also, unless I'm mistaken the `use_ssl` field will be ignored, so we should just remove it.

Reply:
Makes sense and looks cleaner. 

I tried to make the code consistent with it's equivalent code in tutor "core", but messed up the ternary condition.

Reply:
> Also, unless I'm mistaken the `use_ssl` field will be ignored, so we should just remove it.

That seems to be the case, as it is inferred from the scheme in the url.

```
        scheme = kwargs.get("scheme", "http")
        if use_ssl or scheme == "https":
            scheme = "https"
            use_ssl = True
        self.use_ssl = use_ssl
```
from: https://github.com/elastic/elasticsearch-py/blob/df45d85735a0ce7321cf6083637ef43b6bb4fa6a/elasticsearch/connection/base.py#L137-L141